packages/teleport/src/services/history.js

```diff
@@ -124,7 +124,7 @@ export function getUrlParameter(name, path) {
   for (var i = 0; i < vars.length; i++) {
     var pair = vars[i].split('=');
     if (decodeURIComponent(pair[0]) == name) {
-      return decodeURIComponent(pair[1]);
+      return decodeURIComponent(pair[1]).replace(/\+/g, ' ');
```

Comment:
decodeURIComponent should correctly decode the text. If it has + then it should preserve it. The question is why is there "+" in the first place?

Reply:
as disucssed in slack, replaced with URLSearchParams